django/contrib/gis/db/backends/mysql/operations.py

```diff
@@ -29,12 +31,8 @@ def is_mysql_5_6(self):
         return self.connection.mysql_version < (5, 7, 6)
 
     @cached_property
-    def uses_invalid_empty_geometry_collection(self):
```

Comment:
What's your thinking on removing this? I think it might help readability in the test rather than putting the the mysql version check there. I don't know if it might be used elsewhere later.